cmd/auth/describe.go

```diff
@@ -0,0 +1,180 @@
+package auth
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/databricks/cli/cmd/root"
+	"github.com/databricks/cli/libs/cmdio"
+	"github.com/databricks/cli/libs/flags"
+	"github.com/databricks/databricks-sdk-go/config"
+	"github.com/spf13/cobra"
+)
+
+var authTemplate = `{{"Workspace:" | bold}} {{.Details.Host}}
+{{if .Username }}{{"User:" | bold}} {{.Username}}{{end}}
+{{"Authenticated with:" | bold}} {{.Details.AuthType}}
+{{if .AccountID }}{{"Account ID:" | bold}} {{.AccountID}}{{end}}
+-----
+` + configurationTemplate
+
+var errorTemplate = `Unable to authenticate: {{.Error}}
+-----
+` + configurationTemplate
+
+const configurationTemplate = `Configuration:
+  {{- $details := .Details}}
+  {{- range $k, $v := $details.Configuration }}
+  {{if $v.AuthTypeMismatch}}~{{else}}✓{{end}} {{$k | bold}}: {{$v.Value}}
+  {{- if and (not (eq $v.Source.String "dynamic configuration")) $v.Source }}
+  {{- " (from" | italic}} {{$v.Source.String | italic}}
+  {{- if $v.AuthTypeMismatch}}, {{ "not used for auth type " | red | italic }}{{$details.AuthType | red | italic}}{{end}})
+  {{- end}}
+  {{- end}}
+`
+
+func newDescribeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "describe",
+		Short: "Describes which auth credentials is being used by the CLI and identify their source",
```

Comment:
```suggestion
		Short: "Describes the authentication credentials and source being used by the CLI",
```

Reply:
This is hard! Other possibilities:
"Describes the current CLI authentication credentials and their source"
"Describes the authentication credentials that are being used by the CLI and identifies their source"
"Describes the authentication credentials, and their source, being used by the CLI"